src/composables/transactionTx.ts

```diff
@@ -46,17 +47,18 @@ export function useTransactionTx({
   const getPreferred = computed(() => store.getters['names/getPreferred']);
 
   const txType = computed<TransactionType>(() => getTxType.value(innerTx.value));
+  const multisigType = computed(() => innerTx?.value?.function as IMultisigFunctionTypes);
```

Comment:
This is actually wrong because `innerTx?.value?.function` is not related to the multisig. It's a general thing for all transactions, and the type of this is already assigned: `TxFunction`.